client/dashboard/sites/deployments/deployments-list.tsx

```diff
@@ -100,20 +106,46 @@ export function DeploymentsList() {
 
 	const hasFilterOrSearch = ( view.filters && view.filters.length > 0 ) || view.search;
 	const emptyTitle = hasFilterOrSearch ? __( 'No deployments found' ) : __( 'No deployments yet' );
-
+	const deploymentStatusRefreshed =
+		deploymentRuns.find( ( deployment ) => deployment.id === deploymentForModal?.id )?.status ||
+		'unknown';
 	return (
-		<DataViewsCard>
-			<DataViews
-				data={ filteredData }
-				fields={ fields }
-				view={ view }
-				onChangeView={ setView }
-				isLoading={ isLoading }
-				defaultLayouts={ DEFAULT_LAYOUTS }
-				paginationInfo={ paginationInfo }
-				getItemId={ ( item ) => item.id.toString() }
-				empty={ emptyTitle }
-			/>
-		</DataViewsCard>
+		<>
+			<DataViewsCard>
+				<DataViews
+					actions={ [
+						{
+							id: 'open-logs',
+							label: __( 'Open logs' ),
+							isPrimary: true,
+							icon: <Icon icon={ seen } />,
+							callback: ( items ) => {
```

Comment:
We can use the `RenderModal` prop to display the modal, and rename `DeploymentLogsModal` to something like `DeploymentLogsModalContentInfo`, since the `RenderModal` option will automatically wrap the returned element with a `Modal`.

Reply:
Quick note to emphasize that if you use `RenderModal`, you will need to remove the [existing `Modal` wrapper](https://github.com/Automattic/wp-calypso/pull/105747/files/38199f93686c2e1e204cbe0af1bd6a5e11d4c0e7#diff-71a65e44bbafcb5c1a2d8d3a59f1aff038a456c03f8437ea6cb91869bf3ca89eR53-R59) as it will be added by the `Dataviews` component as mentioned above (I did that mistake 😛 )